alembic/versions/526e8250a192_sync_with_db.py

```diff
@@ -0,0 +1,109 @@
+"""sync-with-db
+
+Revision ID: 526e8250a192
+Revises: 367ed257e4ee
+Create Date: 2023-06-15 14:36:57.819815
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from cg.store.models import SampleLaneSequencingMetrics
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = "526e8250a192"
+down_revision = "367ed257e4ee"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column(
+        "application",
+        "prep_category",
+        existing_type=mysql.ENUM("cov", "mic", "rml", "tgs", "wes", "wgs", "wts"),
+        nullable=False,
+    )
+    op.alter_column(
+        "family",
+        "priority",
+        existing_type=mysql.ENUM("research", "standard", "priority", "express", "clinical_trials"),
+        nullable=False,
+    )
+
+    op.alter_column("invoice", "customer_id", existing_type=mysql.INTEGER(), nullable=False)
+
+    # In the db, the foreign key invoice -> customer does not exist, actually. So below would drop the index in the db
+    # and add a foreign key matching the python model. Foreign keys are indexed by default.
+    op.drop_index("customer_invoice_pk", table_name="invoice")
+    op.create_foreign_key(None, "invoice", "customer", ["customer_id"], ["id"])
+    op.create_foreign_key(None, "pool", "invoice", ["invoice_id"], ["id"])
```

Comment:
Very nice! We might need to look trough some code to see that this does not brake anything